### ISSUE TITLE
Remove Emoticons

### ISSUE THREAD

**Issue by WillStrohl** [DNNCommunity/Dnn.CommunityForums#162]
## Is your feature request related to a problem?
There is a feature that allows end-users to optionally select an emoticon while writing their post.  This in itself is a great idea and feature, but it was implemented a long time ago when emoticons were not easily inserted into text.  In today's world, this feature is built into the WYSIWYG editor and when the basic textbox is used, pretty much all keyboards/OSs have an option to allow this.  

## Describe the solution you'd like  
Unless we can get someone to provide a current use case for this feature, we should remove it and all related code/schema that supports it.  

## Describe alternatives you've considered
None at this time.  

## Additional context
None at this time.

**Comment by Timo-Breumelhof** [comment]
As @johnhenley stated before (I think) we should use the CkEditor options / icons IMO.

**Comment by WillStrohl** [comment]
Agreed. This update wouldn't prevent that.  Adding the emoticons is something that the Administrator would do via the text editor configuration and not the Community Forums module, right?

**Comment by johnhenley** [comment]
You can enable emotions in ckeditor and disable in the forums configuration. No sense in maintaining a core editor feature that duplicates functionality IMHO.

**Comment by WillStrohl** [comment]
So to be sure I'm still tracking this properly... We're agreeing that we need to remove the emoticons and keep this Issue open... Right?  :)

**Comment by johnhenley** [comment]
> So to be sure I'm still tracking this properly... We're agreeing that we need to remove the emoticons and keep this Issue open... Right?  :)  

Agreed

**Comment by johnhenley** [comment]
The only caveat is that if you don't use ckeditor you might want to still have it available. Which opens another question of whether ckeditor (or actually "default DNN editor" should just be used everywhere and remove the "active editor" in favor of just being consistent. I think having that option years ago bad sense in low bandwidth scenarios, but does that still apply?
